# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../openvas-plugins/libtool-mode.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN openvas-plugins-0.9.0.orig/plugins/3com_hub/Makefile openvas-plugins-0.9.0/plugins/3com_hub/Makefile
--- openvas-plugins-0.9.0.orig/plugins/3com_hub/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/3com_hub/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -5,8 +5,8 @@
 all : $(PLUGNAME).nes
 	
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include)  -I/usr/local/include `libnet-config --defines --cflags` -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo `libnet-config --defines --libs` -L/usr/local/lib $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include)  -I/usr/local/include `libnet-config --defines --cflags` -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo `libnet-config --defines --libs` -L/usr/local/lib $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/find_service/Makefile openvas-plugins-0.9.0/plugins/find_service/Makefile
--- openvas-plugins-0.9.0.orig/plugins/find_service/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/find_service/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/ftp_write_dirs/Makefile openvas-plugins-0.9.0/plugins/ftp_write_dirs/Makefile
--- openvas-plugins-0.9.0.orig/plugins/ftp_write_dirs/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/ftp_write_dirs/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/hydra/Makefile openvas-plugins-0.9.0/plugins/hydra/Makefile
--- openvas-plugins-0.9.0.orig/plugins/hydra/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/hydra/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : hydra.nes
 hydra.nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o libhydra.la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o libhydra.la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug hydra $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/linux_tftp/Makefile openvas-plugins-0.9.0/plugins/linux_tftp/Makefile
--- openvas-plugins-0.9.0.orig/plugins/linux_tftp/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/linux_tftp/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/nikto_wrapper/Makefile openvas-plugins-0.9.0/plugins/nikto_wrapper/Makefile
--- openvas-plugins-0.9.0.orig/plugins/nikto_wrapper/Makefile	2007-02-22 16:49:41.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/nikto_wrapper/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	$(LIBUTIL) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
diff -ruN openvas-plugins-0.9.0.orig/plugins/nmap_tcp_connect/Makefile openvas-plugins-0.9.0/plugins/nmap_tcp_connect/Makefile
--- openvas-plugins-0.9.0.orig/plugins/nmap_tcp_connect/Makefile	2007-02-22 16:49:43.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/nmap_tcp_connect/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/nmap_wrapper/Makefile openvas-plugins-0.9.0/plugins/nmap_wrapper/Makefile
--- openvas-plugins-0.9.0.orig/plugins/nmap_wrapper/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/nmap_wrapper/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c 
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) $(LIBUTIL) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) $(LIBUTIL) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/objectserver/Makefile openvas-plugins-0.9.0/plugins/objectserver/Makefile
--- openvas-plugins-0.9.0.orig/plugins/objectserver/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/objectserver/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/openvas_tcp_scanner/Makefile openvas-plugins-0.9.0/plugins/openvas_tcp_scanner/Makefile
--- openvas-plugins-0.9.0.orig/plugins/openvas_tcp_scanner/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/openvas_tcp_scanner/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -7,8 +7,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c 
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) -lm $(LIBUTIL) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) -lm $(LIBUTIL) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/smad/Makefile openvas-plugins-0.9.0/plugins/smad/Makefile
--- openvas-plugins-0.9.0.orig/plugins/smad/Makefile	2007-02-22 16:49:41.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/smad/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/snmp_portscan/Makefile openvas-plugins-0.9.0/plugins/snmp_portscan/Makefile
--- openvas-plugins-0.9.0.orig/plugins/snmp_portscan/Makefile	2007-02-22 16:49:42.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/snmp_portscan/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	$(LIBUTIL) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
diff -ruN openvas-plugins-0.9.0.orig/plugins/ssl_ciphers/Makefile openvas-plugins-0.9.0/plugins/ssl_ciphers/Makefile
--- openvas-plugins-0.9.0.orig/plugins/ssl_ciphers/Makefile	2007-02-22 16:49:43.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/ssl_ciphers/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/synscan/Makefile openvas-plugins-0.9.0/plugins/synscan/Makefile
--- openvas-plugins-0.9.0.orig/plugins/synscan/Makefile	2007-02-22 16:49:43.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/synscan/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) $(DEFS) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
diff -ruN openvas-plugins-0.9.0.orig/plugins/tftp_grab_file/Makefile openvas-plugins-0.9.0/plugins/tftp_grab_file/Makefile
--- openvas-plugins-0.9.0.orig/plugins/tftp_grab_file/Makefile	2007-02-22 16:49:43.000000000 +0100
+++ openvas-plugins-0.9.0/plugins/tftp_grab_file/Makefile	2007-09-18 02:43:41.000000000 +0200
@@ -4,8 +4,8 @@
 
 all : $(PLUGNAME).nes
 $(PLUGNAME).nes : $(PLUGNAME).c
-	$(LIBTOOL) $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
-	$(LIBTOOL) $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
+	$(LIBTOOL) --mode=compile --tag CC $(CC) $(DEFS) $(include) -c  $(PLUGNAME).c
+	$(LIBTOOL) --mode=link --tag CC $(CC) -o lib$(PLUGNAME).la $(PLUGNAME).lo $(LIBS) \
 	-rpath $(rootdir)/bin/plugins
 	../install_plug $(PLUGNAME) $(rootdir)
 clean :
